apps/server/src/trpc/routes/entityCreate.test.ts

```diff
@@ -68,3 +68,28 @@ test("updates existing entity with new type", async ({
   expect(brand.id).toEqual(entity.id);
   expect(brand.type).toEqual(["distiller", "brand"]);
 });
+
+test("creates entity with parent", async ({ defaults, fixtures }) => {
```

Comment:
add another test to create an entity as an entirely new nested object, vs passing it as an id

Reply:
You still need to add this additional test